Dashboard/app/js/lib/views/maps/map-views-common.js

```diff
@@ -660,21 +707,39 @@ FLOW.NavMapsView = FLOW.View.extend({
                 if(questionAnswer !== "" && questionAnswer !== null && questionAnswer !== "null"){
                   switch (self.questions[i].questionType) {
                     case "PHOTO":
-                      var imageString = "", imageJson;
+                    case "VIDEO":
+                      var mediaString = "", mediaJson, mediaFilename = "", mediaObject = {};
                       if (questionAnswer.charAt(0) === '{') {
-                        imageJson = JSON.parse(questionAnswer);
-                        imageString = imageJson.filename
+                        mediaJson = JSON.parse(questionAnswer);
+                        mediaString = mediaJson.filename;
+
+                        if(mediaJson.location != null) {
+                          mediaObject['filename'] = mediaString.substring(mediaString.lastIndexOf("/")+1);
+                          mediaObject['lat'] = mediaJson['location']['latitude'];
+                          mediaObject['lng'] = mediaJson['location']['longitude'];
+                          mediaResponses.push(mediaObject);
+                          mediaFilename = mediaObject['filename'].split(".")[0];
+                        }
                       } else {
-                        imageString = questionAnswer;
+                        mediaString = questionAnswer;
                       }
 
-                      var image = '<div class=":imgContainer photoUrl:shown:hidden">';
-                      var imageFilename = FLOW.Env.photo_url_root+imageString.substring(imageString.lastIndexOf("/")+1);
-                      image += '<a href="'+imageFilename+'" target="_blank">'
-                      +'<img src="'+imageFilename+'" alt=""/></a>';
-
-                      image += '</div>';
-                      clickedPointContent += image;
+                      var mediaFileURL = FLOW.Env.photo_url_root+mediaString.substring(mediaString.lastIndexOf("/")+1);
+                      if(self.questions[i].questionType == "PHOTO") {
+                        mediaOutput = '<div class=":imgContainer photoUrl:shown:hidden">'
+                        +'<a class="media" data-coordinates=\''
+                        +((typeof mediaObject.lat !== 'undefined') ? JSON.stringify(mediaObject) : '' )+'\' href="'
```

Comment:
@muloem since not all media is always geotagged, some objects will lack the coordinates properties (lat and lng). The typeof check prevents an undefined error being thrown in such cases